backend/middleware/sanitize.test.ts

```
@@ -181,11 +175,11 @@ describe('sanitizeWithSchema', () => {
       b: { sanitize: true, mode: 'plain' },
       c: { sanitize: true, mode: 'rich' },
     };
-    const result = sanitizeWithSchema(input, schema) as Record<string, unknown>;
-    expect(result['a']).toBe('');
-    expect(result['b']).not.toContain('<script>');
-    expect(result['c']).toContain('<b>𝓍𝓈𝓈</b>');
-    expect(result['c']).not.toContain('\u200B');
+    const result = sanitizeWithSchema(input, schema);
+    expect((result as any).a).toBe('');
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)